tests/unit_test/data_processor/upload_handler_log_processor_test.py

```
@@ -0,0 +1,146 @@
+        assert log[0]['collection_name']
+        assert log[0]['upload_errors']["File type error"] == "Invalid file type"
+        assert log[0]["status"] == "Failure"
+        assert log[0]['end_timestamp'] is not None
+        assert log[0]['start_timestamp'] is not None
+
+    def test_is_limit_exceeded_exception(self, monkeypatch):
+        bot = 'test_bot'
+        try:
+            bot_settings = BotSettings.objects(bot=bot).get()
+            bot_settings.system_limits["file_upload_limit"] = 0
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)